test/integration/apiserver/apply/reset_fields_test.go

```diff
@@ -57,7 +57,6 @@ var resetFieldsStatusData = map[schema.GroupVersionResource]string{
 	gvr("networking.k8s.io", "v1", "ingresses"):                     `{"status": {"loadBalancer": {"ingress": [{"ip": "127.0.0.2"}]}}}`,
 	gvr("autoscaling", "v1", "horizontalpodautoscalers"):            `{"status": {"currentReplicas": 25}}`,
 	gvr("batch", "v1", "cronjobs"):                                  `{"status": {"lastScheduleTime":  "2020-01-01T00:00:00Z"}}`,
-	gvr("batch", "v1beta1", "cronjobs"):                             `{"status": {"lastScheduleTime":  "2020-01-01T00:00:00Z"}}`,
```

Comment:
I'd leave these for now, they are ensuring we properly react with batch/v1beta1.